xep-0065.xml

```diff
@@ -271,6 +281,32 @@
     <feature var='http://jabber.org/protocol/bytestreams'/>
   </query>
 </iq>
+]]></example>
+  <p>Alternatively, the proxy may be exposed directly on the Requester’s server. In which case, the Requester should use Service Discovery on this domain, like for every previous disco#items results.</p>
```

Comment:
nit: a quick grep in the XEPs repo gives the impression that 'requester' is usually written lowercase